pages/DSNP/Announcements.md

```diff
@@ -22,7 +22,7 @@ Each Announcement has a enumerated type for use when separating out a stream of
 | 3 | [Reply](Types/Reply.md) | a public response to a Broadcast | YES | YES |
 | 4 | [Reaction](Types/Reaction.md) | a public visual reply to a Broadcast | no | YES |
 | 5 | [Profile](Types/Profile.md) | a profile | YES | no |
-
+| 6 | [Update](Types/Update.md) | an update to content| YES | YES |
```

Comment:
Should Updates be tombstonable? Or is it just replaced with another edit?

Reply:
I can see a scenario where a user might want to Tombstone an update as a way of saying "I'd like to undo this". I think it should be possible.

Reply:
That's a nested search however. Whereas a second `Update` announcement would be found in the first search and performs the same result right?

Reply:
Just to be sure, what do we mean by "nested search"? If you mean that a Tombstone will force a client/listening node to do a lookup for the content to nullify, wouldn't that be true for another Update as well?

Another question -- is it possible that a user would want to nullify an Update as opposed to editing it with another Announcement? If both use cases are valid, my opinion is that we should allow it at the spec level and assume that developers implementing the spec will handle the exact implementation. Thoughts?

Reply:
Notes from the weekly spec discussion 2022-04-21

- Updates should NOT be able to be Tombstoned, just the original.
- If the original is Tombstoned, Update Announcements that target it are ignored

```suggestion
| 6 | [Update](Types/Update.md) | an update to content| YES | no |
```

That suggestion brings up the question of if Update Announcements have a DSNP Content URI. This line says yes, but I could see how we actually don't want it to be considered a new DSNP Content URI. Thoughts?

Reply:
Noting for posterity that we decided _not_ to allow tombstoning updates.